graphics/unpaper/Portfile

```diff
@@ -2,43 +2,43 @@
 
 PortSystem          1.0
 PortGroup           github 1.0
+PortGroup           meson 1.0
 
-# When updating to the next version, please switch to xz tarball
-# and no longer autoreconf.
-github.setup        Flameeyes unpaper 6.1 unpaper-
-revision            2
+github.setup        unpaper unpaper 7.0.0 unpaper-
+github.tarball_from releases
+use_xz              yes
+
+name                unpaper
+version             7.0.0
```

Comment:
name and version are set already using the github.setup line; please do not duplicate

Reply:
ahh of course, thanks